pkg/controllers/resources/storageclasses/syncer_test.go

```diff
@@ -32,7 +32,7 @@ func TestSync(t *testing.T) {
 			Name:            translate.Default.HostNameCluster(vObjectMeta.Name),
 			ResourceVersion: syncertesting.FakeClientResourceVersion,
 			Labels: map[string]string{
-				translate.MarkerLabel: translate.VClusterName,
+				translate.MarkerLabel: translate.SafeConcatName(testingutil.DefaultTestTargetNamespace, "x", translate.VClusterName),
```

Comment:
Previous check (`translate.MarkerLabel: translate.VClusterName`) was incorrect, but since the tests were stripping down and not checking the `vcluster.loft.sh/managed-by` label, the was not catching the error in the test here.